stories/utils/StoryFragment.tsx

```diff
@@ -0,0 +1,15 @@
+import "../../src/ui/reference/_colors.scss";
```

Comment:
I couldn't figure out the purpose of this component 🤔

Reply:
Otherwise default css variables within _colors or _measurement were not showing up on Storybook. So its a bit of a hack